### ISSUE TITLE
Add link to 1.0 README - code snippets on master are not valid with 1.0.0

### ISSUE THREAD

**Issue by alexkhimich** [cloudevents/sdk-javascript#174]
This snippet 
```
const {
  CloudEvent,
  HTTPReceiver
} = require("cloudevents-sdk");

// Create a receiver to accept events over HTTP
const receiver = new HTTPReceiver();

// body and headers come from an incoming HTTP request, e.g. express.js
const receivedEvent = receiver.accept(req.body, req.headers);
console.log(receivedEvent.format());
```
throws 

```

    TypeError: HTTPReceiver is not a constructor

      2 | 
      3 | // Create a receiver to accept events over HTTP
    > 4 | const receiver = new HTTPReceiver();
```

**Comment by lance** [comment]
@alexkhimich - I imagine you are using the published 1.0.0 version of the module. The API is rapidly changing, and so what is on the README is ahead of what's available on npmjs.com. I agree this is a problem. In the meantime, you might look at the 1.0.0 version of the README here https://github.com/cloudevents/sdk-javascript/blob/v1.0.0/README.md#how-to-use. Apologies.

**Comment by lance** [comment]
2.0.0 released today
